FestivalPlanner/films/forms/film_forms.py

```diff
@@ -43,29 +46,64 @@ class PickRating(forms.Form):
     @classmethod
     def update_rating(cls, session, film, fan, rating_value, post_attendance=False):
         field = FIELD_BY_POST_ATTENDANCE[post_attendance]
-        manager = MANAGER_BY_POST_ATTENDANCE[post_attendance]
         old_rating_str = fan_rating_str(fan, film, post_attendance=post_attendance)
+        new_rating = None
 
-        # Update the indicated rating.
-        new_rating, created = manager.update_or_create(
-            film=film,
-            film_fan=fan,
-            defaults={field: rating_value},
-        )
-
-        # Remove zero-ratings (unrated).
-        kwargs = {'film': film, 'film_fan': fan, field: 0}
-        zero_ratings = manager.filter(**kwargs)
-        if len(zero_ratings) > 0:
-            zero_ratings.delete()
+        # Account for possible alternative titles.
+        alternative_films = cls.get_alternative_films(film, post_attendance)
+        for alt_film in alternative_films:
+            new_rating = cls.update_one_rating(session, alt_film, fan, rating_value, post_attendance)
 
```

Comment:
Please change names as to make clear that the normal case, no alternative titles, is included here.